src/views/summeruniversity/BoardView.vue

```diff
@@ -59,10 +59,15 @@
             </b-table-column>
 
             <b-table-column field="status" label="Status" sortable>
-              <span v-if="props.row.status === 'accepted'">Accepted</span>
-              <span v-if="props.row.status === 'rejected'">Rejected</span>
-              <span v-if="props.row.status === 'waiting_list'">Waiting list</span>
-              <span v-if="props.row.status === 'pending'">Pending</span>
+              <span v-if="props.row.cancelled">
+                <span class="tag is-danger" v-if="props.row.cancelled">
```

Comment:
This looks strange for rejected applications. And it would be better if cancelled participants do not have a colored column (or colored grey)
![image](https://github.com/AEGEE/frontend/assets/13023439/2ce7e729-1611-4a7f-818d-8aae270eab7e)